api/src/main/java/uk/gov/dvsa/mot/vehicle/read/core/VehicleReadService.java

```diff
@@ -14,6 +15,8 @@
 
     List<Vehicle> findByRegistration(String registration);
 
+    List<DvlaVehicle> findDvlaVehicleByRegistration(String registration);
```

Comment:
`DvlaVehicle` is the persist model - this class represents the objects from database. We should return `api` model, something like in `uk.gov.dvsa.mot.vehicle.api`.